awx/main/models/ha.py

```diff
@@ -152,6 +156,11 @@ def get_absolute_url(self, request=None):
     def capacity(self):
         return sum([inst.capacity for inst in self.instances.all()])
 
+    def _get_active_jobs(self):
+        return UnifiedJob.objects.filter(instance_group=self,
```

Comment:
It is not obvious that this method only exists to satisfy `RelatedJobsMixin`. It would be nice to have a commet, mayb something like `'''Implements RelatedJobsMixin._get_active_jobs'''`